serializers/postgres.rb

```diff
@@ -27,7 +27,11 @@ def self.serialize_internal(pg, options = {})
       )
 
       if pg.timeline && pg.representative_server&.primary?
-        base[:earliest_restore_time] = pg.timeline.earliest_restore_time&.utc&.iso8601
+        begin
+          base[:earliest_restore_time] = pg.timeline.earliest_restore_time&.utc&.iso8601
+        rescue => ex
```

Comment:
we probably need a test to cover this.

Reply:
Yes, initially it was not required since the rescue part is empty, but now we need a test. I added it now.